server/rest/annotation.py

```diff
@@ -92,14 +92,41 @@ def getAnnotationSchema(self, params):
     @describeRoute(
         Description('Get an annotation by id.')
         .param('id', 'The ID of the annotation.', paramType='path')
+        .param('left', 'The left column of the area to fetch.',
+               required=False, dataType='float')
+        .param('right', 'The right column (exclusive) of the area to fetch.',
+               required=False, dataType='float')
+        .param('top', 'The top row of the area to fetch.',
+               required=False, dataType='float')
+        .param('bottom', 'The bottom row (exclusive) of the area to fetch.',
+               required=False, dataType='float')
+        .param('low', 'The lowest z value of the area to fetch.',
+               required=False, dataType='float')
+        .param('high', 'The highest z value (exclusive) of the area to fetch.',
+               required=False, dataType='float')
+        .param('minimumSize', 'Only annotations larger than or equal to this '
+               'size in pixels will be returned.  Size is determine by the '
+               'minimum orthogonal extent of the annotation.  This probably '
```

Comment:
I changed what I was using for the size but failed to update this text.  Originally I was using the minimum orthogonal extent, but thin elements were undervalued.